c/Cairo/build_info.json

```diff
@@ -0,0 +1,15 @@
+{
+    "maintainer": "stutiibm",
+    "package_name": "Cairo",
```

Comment:
Lets keep first letter of package name and folder as small case only
folder name expected to be all small cases.
@stutiibm